db/migrate/20210805135643_remove_insert_size_from_pacbio_run_well.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveInsertSizeFromPacbioRunWell < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
need to think about this migration as not sure how it is going to work with legacy data, if fragment size is already populated in pools it might be fine, but then it may not be same values as previous insert_size on old runs